x/auth/ante/sigverify.go

```diff
@@ -269,7 +269,7 @@ func (svd SigVerificationDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simul
 		}
 
 		// retrieve signer data
-		genesis := ctx.BlockHeight() == 0
```

Comment:
Lets remove this `||` here, and just use `IsGenesis`

Reply:
or alternatively do `IsGenesis && ctx.BlockHeight() == ctx.InitialHeight()` as a defense in depth.

Reply:
> Lets remove this || here, and just use IsGenesis

Too many tests fail that would require funky re-wiring. This is why I suggested to add the `||` to keep the tests passing. Probably because `IsGenesis` returns false as it does not go through the `InitGenesis` flow.

Alternatively, the `|| ctx.BlockHeight() == ctx.InitialHeight()` can work too.